resources/views/content/styles.twig

```diff
@@ -1,2 +1,2 @@
 <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css" />
-<link rel="stylesheet" href="{{ plugin_path("Findologic") }}/css/findologic.css" />
\ No newline at end of file
+<link rel="stylesheet" href="{{ plugin_path("Findologic") }}/css/findologic.min.css" />
```

Comment:
As described in the story, `findologic.min.css` is supposed to be loaded additionally, not instead of `findologic.css`.